src/main/java/frc/robot/constants/drivetrain/DrivetrainConstants.java

```diff
@@ -8,4 +17,17 @@ public class DrivetrainConstants
     /** Radians per second */
     public static final double MAX_ANGULAR_RATE = Math.PI;
 
+        public static final AprilTagFieldLayout TAG_LAYOUT;
+        
+        static {
+            try {
+                TAG_LAYOUT = AprilTagFields.kDefaultField.loadAprilTagLayoutField();
+            } catch (UncheckedIOException e) {
+                e.printStackTrace();
+                throw new RuntimeException(e);
+            }
+        }
+
+        public static final double AMBIGUITY_THRESHOLD = 0.25;
+        public static final Transform3d ROBOT_TO_CAM = new Transform3d(new Translation3d(Units.inchesToMeters(6.676595), 0.0, Units.inchesToMeters(27.63304)), new Rotation3d(0, -45, 0));
```

Comment:
create constants for both cameras even if we aren't using it yet. Create unique names (ie. FORWARD_CAMERA_POSITION or REAR_CAMERA_POSITION)